### ISSUE TITLE
[media-kit] Pin media-gfx/inkscape-1.3.2 (inkscape-1.4 fails to build)

### ISSUE THREAD

**Issue by aardwolfweb** [macaroni-os/mark-issues#231]
### TL; DR:
Pin `inkscape-1.3.2`.

### Background
`inkscape-1.4` was recently released. The patch included in `mark-testing` is leftover from `inkscape-1.3.2` and patching fails. Build fails without the patch.

I played around with the `1.4` patches that Gentoo is using but didn't have any luck, possibly due to them being for newer versions of `poppler`. Gentoo seems to be on `poppler-24.11.0` with an additional backport patch for `poppler-24.10`. `mark-testing` is on `poppler-24.04.0`. (The newer `poppler` may be needed for the font editing features in `inkscape-1.4` - that's where the compile seems to be failing. Or possibly, just in the general font handling.)

After a discussion on Discord, the consensus was that updating `poppler` is probably necessary for `inkscape-1.4` to build. However, since it touches so many other packages, it's likely to be a big job and the `openssl` version update (Also, a big job.) is a better use of developer time, especially due to it being a security update.

The conclusion, was to pin `inkscape-1.3.2` for now.
